File: stock-market-service/src/main/java/rs/edu/raf/dto/StockDTO.java
package rs.edu.raf.dto;

public record StockDTO (String ticker,
                        String description,
                        String exchange,
                        Long lastRefresh,
                        Double price,
                        Double high,
                        Double low,
                        Double change,
                        Long volume,
                        Long contractSize,
                        Double maintenanceMargin,
                        Double changePercent,
                        Double dollarVolume,
                        Double nominalValue,
                        Double initialMarginCost,
                        Long outstandingShares,
                        Double dividendYield,
                        Double marketCap) {
}

File: stock-market-service-kotlin/src/main/kotlin/org/example/stockmarketservicekotlin/dto/StockDTO.kt
package org.example.stockmarketservicekotlin.dto

data class StockDTO(
    val ticker: String,
    val description: String,
    val exchange: String,
    val lastRefresh: Long,
    val price: Double,
    val high: Double,
    val low: Double,
    val change: Double,
    val volume: Long,
    val contractSize: Long,
    val maintenanceMargin: Double,
    val changePercent: Double,
    val dollarVolume: Double,
    val nominalValue: Double,
    val initialMarginCost: Double,
    val outstandingShares: Long,
    val dividendYield: Double,
    val marketCap: Double
)
